### PR TITLE
kbdd: init

### PR DESCRIPTION
**Pull request by wedens**

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -12350,6 +12350,8 @@ let
 
   sxhkd = callPackage ../applications/window-managers/sxhkd { };
 
+  kbdd = callPackage ../applications/window-managers/kbdd { };
```
Comment (cillianderoiste) [inline pkgs/top-level/all-packages.nix]: The packages are supposed to be sorted alphabetically, please move it up.

File: pkgs/applications/window-managers/kbdd/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    rev = "47dee0232f157cd865e43d92005a2ba107f6fd75";
+    sha256 = "1b9a66d216326a9759cad26393fbf8259fe7a0c2dd1075047fc989f0e52d969f";
+  };
+
+  buildInputs = [ pkgconfig xorg.libX11 dbus_glib autoreconfHook ];
+
+  meta = {
+    description = "Simple daemon and library to make per window layout using XKB";
+    homepage = https://github.com/qnikst/kbdd;
+    license = stdenv.lib.licenses.gpl3;
```
Comment (cillianderoiste) [inline pkgs/applications/window-managers/kbdd/default.nix]: Please add yourself as the maintainer.

**Comment by wedens** [comment]
@cillianderoiste done

**Comment by cillianderoiste** [comment]
Excellent, thanks!

**Pull request merged**